slack-server-integration-common/src/main/resources/com/atlassian/plugins/slack/slack-common.properties

```diff
@@ -131,7 +131,7 @@ plugins.slack.admin.connection.user.status.oauth.failure.tooltip.description=You
   <a href="#" class="slack-user-link">update access to your Slack account</a>.
 
 slack.oauth2.view.oauth.sessions.page.title=Slack accounts
-slack.oauth2.view.oauth.sessions.page.text=To post notifications in a Slack workspace, {0} needs to be integrated with the workspace\
+slack.oauth2.view.oauth.sessions.page.text=To post notifications in a Slack workspace, {0} needs to be integrated with the workspace \
```

Comment:
This one is already present in `dev` after merging #239 . Please rebase your dev branch to the latest `dev` in order to get rid of this change.

Reply:
Ok